### ISSUE TITLE
Persist party_teambased settings.

### ISSUE THREAD

**Issue by syaifulnizamyahya** [XLabsProject/iw4x-client#103]
Party server wont start match if there is only 1 person in the lobby. I had to manually set party_teambased 1 in order for the match to start. Since this value is resetted everytime lobby is loaded, I had to manually set party_teambased 1 again.

Is it possible to persist the value set for party_teambased?

**Comment by diamante0018** [comment]
I don't see an advantage in locking this dvar to 1. It is used by the game when it processes the Playlist Rules.

**Comment by xerxes-at** [comment]
I think you mixed up 0 and 1, I remember you from Discord.
<details>
<summary> Topic on Discord</summary>
Is it possible to create a lobby server that starts the game with 1 player?
Tried editing *.info file to have this rule party_minplayers 1, but when joining the server, in client it says waiting for balancing team. game never start
Tried scr_teambalance 0 still same
publicENEMY — Yesterday at 2:49 PM
lobby_partySearchWaitTime 60 still same after waiting for more than 1 minute
The lobby switching between these 2 state. Waiting for more players to balance teams... and The match is about to begin.
publicENEMY — Yesterday at 3:01 PM
This is what I have in logs. Endless of this. 
[     88478] Gamestate changing due to gameStartTime being set in PREMATCH state
[     88483] Gamestate changing due to lobbyState changing to STARTREADY
[     88485] Gamestate changing due to lobbyState changing to WAITING
[     88494] Gamestate changing due to gameStartTime being set in PREMATCH state
[     88500] Gamestate changing due to lobbyState changing to STARTREADY
[     88503] Gamestate changing due to lobbyState changing to WAITING
[     88510] Gamestate changing due to gameStartTime being set in PREMATCH state
[     88516] Gamestate changing due to lobbyState changing to STARTREADY
[     88518] Gamestate changing due to lobbyState changing to WAITING
COPY
RaidMax — Yesterday at 3:02 PM
Just make a regular server not a party server?
publicENEMY — Yesterday at 3:04 PM
it works if i change set PlaylistGameMode=3 (ffa). but i want to play ground war playlist.
publicENEMY — Yesterday at 3:05 PM
I have successfully create a regular server. I just want to try party server. For the ability to vote map in gui.
publicENEMY — Yesterday at 3:26 PM
Got it work. Have to set party_teambased 0

publicENEMY — Yesterday at 4:34 PM
party_teambased is reseted everytime we go to lobby
is there a way to force party_teambased to 0 permanently?
RaidMax — Yesterday at 4:36 PM
probably with a mod
or set party_teambased 0 in the start config/script
publicENEMY — Yesterday at 4:53 PM
I add this to the start of batch file. +set party_teambased "0". Works for the first time. Next match it resets back to 1.
I add rule party_teambased 0 to every playlist and global variables. doesnt work.
it seems that there is some script is loaded when opening lobby
</details>

This however is a non issue, you are trying to run a party server on the Ground War playlist which is a team based game mode and having the game set the dvar to 1 is it's expected behavior. You could try to change the flags of the dvar to read only and the default value to 0 or to directly hook where it sets it and skip it this however is no trivial task that a normal user can do.

**Comment by syaifulnizamyahya** [comment]
Yes, I did ask this in Discord. 
When lobby is loaded, the value depends on playlist. If its ffa, its party_teambased 1. If its Ground War, its party_teambased 2. How can I set party_teambased 1 if its Ground War when entering lobby?

The steps that you mentioned, Im not sure how to do it. I believe its not simply setting the value in console.

Is there a way simple enough for non technical user to do?

Thanks.
